conan/tools/cmake/cmakedeps2/target_configuration.py

```diff
@@ -456,4 +479,13 @@ def _template(self):
         set_target_properties({{exe}} PROPERTIES IMPORTED_LOCATION_{{config}} "{{location}}")
         set_property(TARGET {{exe}} PROPERTY CONAN_CONTEXT "{{context}}")
         {% endfor %}
+
+        {% if sources %}
+        ################# Sources information ##############
+        {% for target, srcs in sources.items() %}
+        #################### {{target}} ####################
+        set_property(TARGET {{target}} APPEND PROPERTY INTERFACE_SOURCES
+                     {{config_wrapper(config, srcs | join(';') )}})
```

Comment:
Please change one of the functional tests to use more than 1 source file to validate this join with ;

Reply:
done!